docs/installation/kepler-operator.md

```diff
@@ -1,22 +1,98 @@
 # Kepler Operator on Kind
 
-## Getting Started
+## Requirements:
 
-You’ll need a Kubernetes cluster to run against. You can use KIND to get a local cluster for testing, or run against a remote cluster. Note: Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster kubectl cluster-info shows).
+Before you start make sure you have:
 
-### To run a kind cluster locally
+- `kubectl` installed
+- Clone the `kepler-operator`[repository](https://github.com/sustainable-computing-io/kepler-operator)  
+- You’ll need a Kubernetes cluster to run against. You can use KIND to get a [local cluster for testing](#run-a-kind-cluster-locally), or run against a remote cluster. Note: Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster kubectl cluster-info shows).
+- Signed in as `kubeadmin` or a user with `cluster-admin` role
+
+
+
+### Run a kind cluster locally
 
 ``` sh
+cd kepler-operator
 make cluster-up CLUSTER_PROVIDER='kind' CI_DEPLOY=true GRAFANA_ENABLE=true
```

Comment:
Offtopic: 
Does `CI_DEPLOY` and `GRAFANA_ENABLE` have any impact since IIUC are not passed to local-dev-cluster main.sh??